File: Samples/Digital signatures/SignByMultiplePeople/VB.NET/Credentials.vb
Public Class Credentials
    Public ReadOnly Name As String
    Public ReadOnly Keystore As String
    Public ReadOnly Password As String

    Public Sub New(name As String, keystore As String, password As String)
        Me.Name = name
        Me.Keystore = keystore
        Me.Password = password
    End Sub
End Class

File: Samples/Digital signatures/SignByMultiplePeople/C#/Credentials.cs
namespace SignByMultiplePeople
{
    internal readonly record struct Credentials
    {
        public readonly string Name;
        public readonly string Keystore;
        public readonly string Password;

        public Credentials(string name, string keystore, string password)
        {
            Name = name;
            Keystore = keystore;
            Password = password;
        }
    }
}
